dotnet/src/webdriver/BiDi/Modules/Script/LocalValue.cs

```diff
@@ -160,9 +160,10 @@ public record BigIntLocalValue(string Value) : PrimitiveProtocolLocalValue;
 
 public record ChannelLocalValue(ChannelProperties Value) : LocalValue
 {
-    // TODO: Revise why we need it
+    // AddPreloadScript takes arguments typed as ChannelLocalValue but still requires "type":"channel"
     [JsonInclude]
-    internal string type = "channel";
+    [JsonPropertyName("type")]
```

Comment:
Unnecessary attribute, in my head we will just rename property rather than support backward compatibility. Because if we will not rename properties according spec (even if it is breaking change), we will become to a chaos where the .NET types mess weakly correlated to spec.